autoscheduler/autoscheduler/config/config.py

```diff
@@ -0,0 +1,23 @@
+import os
+import json
+
+
+def get_prop(prop):
+    """
+    get property from postgres-info.json, or create that file if needed
+    """
+    infoFile = os.path.dirname(__file__) + "/postgres-info.json"
```

Comment:
```suggestion
    info_file= os.path.dirname(__file__) + "/postgres-info.json"
```
Rename to follow snake_case naming style

Reply:
fixed